api/app/signals/apps/api/v1/filters.py

```diff
@@ -27,6 +27,16 @@
 )
 
 
+def area_type_choices():
+    options = AreaType.objects.values_list('code', 'name')
+    return [(c, f'{n} ({c})') for c, n in options]
+
+
+def area_choices():
+    options = Area.objects.values_list('code', '_type__name')
```

Comment:
Why not show the name of the `Area` here? In Amsterdam the boroughs have area type SIA Stadsdeel (code `'sia-stadsdeel'`) and area names like `'Zuid'` (code). This wil result in both filters having the same human readable names.

Reply:
we can have duplicate names in areas